lib/rubocop/cop/lint/binary_operator_with_identical_operands.rb

```diff
@@ -27,15 +28,19 @@ module Lint
       #     left_child || left_child
       #   end
       #
+      #   # good
+      #   x + x
+      #   1 << 1
+      #
       class BinaryOperatorWithIdenticalOperands < Base
         MSG = 'Binary operator `%<op>s` has identical operands.'
-        MATH_OPERATORS = %i[+ - * / % ** << >> | ^].to_set.freeze
+        MATH_OPERATORS = %i[+ * ** << >>].to_set.freeze
```

Comment:
I think this name is now a bit misleading, as it doesn't relay the fact those are the ops that are considered fine.

Reply:
Agreed. Updated.